pkgs/data/icons/la-capitaine-icon-theme/default.nix

```
@@ -0,0 +1,53 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, breeze-icons
+, gnome-icon-theme
+, hicolor-icon-theme
+, pantheon
+}:
+
+stdenv.mkDerivation rec {
+  pname = "la-capitaine-icon-theme";
+  version = "0.6.2";
+
+  src = fetchFromGitHub {
+    owner = "keeferrourke";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0id2dddx6rl71472l47vafx968wnklmq6b980br68w82kcvqczzs";
+  };
+
+  propagatedBuildInputs = [
+    breeze-icons
+    gnome-icon-theme
+    hicolor-icon-theme
+    pantheon.elementary-icon-theme
+  ];
+
+  dontDropIconThemeCache = true;
+
+  postPatch = ''
+    patchShebangs configure
+
+    substituteInPlace configure --replace \
+      'DISTRO=$(format "$(lsb_release -si 2>/dev/null)")' \
+      'DISTRO=nixos'
```

```suggestion
    substituteInPlace configure \
      --replace 'DISTRO=$(format "$(lsb_release -si 2>/dev/null)")' 'DISTRO=nixos'
```
This is a more stamdard formatting

---

Done.